frontend/package-lock.json

```diff
@@ -6885,114 +6979,249 @@
+      "license": "Apache-2.0",
+      "dependencies": {
+        "axios": "^1.7.2",
+        "tslib": "^2.6.2"
+      }
+    },
+    "node_modules/@redhat-cloud-services/javascript-clients-shared/node_modules/axios": {
+      "version": "1.10.0",
+      "resolved": "https://registry.npmjs.org/axios/-/axios-1.10.0.tgz",
+      "integrity": "sha512-/1xYAC4MP/HEG+3duIhFr4ZQXR4sQXOIe+o6sdqzeykGLx6Upp/1p8MHqhINOvGeP7xyNHe7tsiJByc4SSVUxw==",
+      "license": "MIT",
+      "dependencies": {
+        "follow-redirects": "^1.15.6",
+        "form-data": "^4.0.0",
+        "proxy-from-env": "^1.1.0"
+      }
     },
     "node_modules/@redhat-cloud-services/rbac-client": {
-      "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/@redhat-cloud-services/rbac-client/-/rbac-client-1.4.0.tgz",
-      "integrity": "sha512-gFwvrOi0MDYbRYwC5TZ4zff4QO/Ss0T+LngWiEYD5duNAEWfbNQft4i/RjvJwEVJ14lxz1xGiHU6WRHms7GeBQ==",
+      "version": "2.2.11",
+      "resolved": "https://registry.npmjs.org/@redhat-cloud-services/rbac-client/-/rbac-client-2.2.11.tgz",
+      "integrity": "sha512-3LLW5XtBMqcdlYJ4lAFo7MaGnHXUBP01O3kty4ocU69EPjiip6/Gtj83vAx5hkgAWX9h6ZEntTDeDuU/ZfGdKQ==",
+      "license": "Apache-2.0",
       "dependencies": {
-        "axios": "^0.27.2",
+        "@redhat-cloud-services/javascript-clients-shared": "^1.2.4",
+        "axios": "^1.7.2",
         "tslib": "^2.6.2"
       }
     },
+    "node_modules/@redhat-cloud-services/rbac-client/node_modules/axios": {
+      "version": "1.10.0",
+      "resolved": "https://registry.npmjs.org/axios/-/axios-1.10.0.tgz",
+      "integrity": "sha512-/1xYAC4MP/HEG+3duIhFr4ZQXR4sQXOIe+o6sdqzeykGLx6Upp/1p8MHqhINOvGeP7xyNHe7tsiJByc4SSVUxw==",
+      "license": "MIT",
+      "dependencies": {
+        "follow-redirects": "^1.15.6",
+        "form-data": "^4.0.0",
+        "proxy-from-env": "^1.1.0"
+      }
+    },
     "node_modules/@redhat-cloud-services/rule-components": {
-      "version": "3.2.7",
-      "resolved": "https://registry.npmjs.org/@redhat-cloud-services/rule-components/-/rule-components-3.2.7.tgz",
-      "integrity": "sha512-NRjZhJ3WXOnkLgEOqCvOeAi/04nVB1QvNQ/Aj9zor7d65E/GtjU+2pfYOqiU/ryTpfTy2bqVuUOefUoHjXqIiw==",
-      "deprecated": "Do not use the version 3.7.2",
+      "version": "4.0.10",
+      "resolved": "https://registry.npmjs.org/@redhat-cloud-services/rule-components/-/rule-components-4.0.10.tgz",
+      "integrity": "sha512-8qht2qVABbtW6Pwo+ytKJXcdjapvnYfDNwnvFJJrSePwsptrpzyHEBguNi0oMpR6m3lJDewI8Y8i9MWRyg/h7Q==",
+      "license": "Apache-2.0",
       "dependencies": {
-        "@patternfly/react-core": "^5.0.0",
-        "@patternfly/react-icons": "^5.0.0",
-        "@patternfly/react-table": "^5.0.0",
-        "@redhat-cloud-services/frontend-components": "^4.0.0",
-        "@redhat-cloud-services/frontend-components-utilities": "^4.0.0",
+        "@patternfly/react-core": "^6.0.0",
```

Comment:
I'm concerned that upgrading @redhat-cloud-services/rule-components is introducing a dependency on PF6.

I don't think the display would necessarily be correct on older OpenShift versions.